sdk/monitor/Azure.Monitor.Ingestion/src/LogsIngestionClient.cs

```diff
@@ -37,6 +41,20 @@ public BatchedLogs(List<T> logsList, BinaryData logsData)
             public BinaryData LogsData { get; }
         }
 
+        internal readonly struct BatchUploadTask
```

Comment:
The part that I found confusing about the name is that it has "Task" in it so you end up having `Task<BatchUploadTask>`.

Reply:
oh I was trying to indicate it was used for the concurrency - maybe this just makes it more confusing